ocs_ci/ocs/registry.py

```diff
@@ -172,6 +165,25 @@ def add_role_to_user(role_type, user):
     logger.info(f"Role_type {role_type} added to the user {user}")
 
 
+def remove_role_from_user(role_type, user):
+    """
+    Function to remove role to user
+
+    Args:
+        role_type (str): Type of the role to be removed
+        user (str): User of the role
+
+    Raises:
+        AssertionError: When failure in removing role from user
+
+    """
+    ocp_obj = ocp.OCP()
+    role_cmd = f"policy remove-role-from-user {role_type} {user} " \
```

Comment:
please break lines with parentheses

Reply:
please take care of this in follow on PR when you update the testcase next time.

Reply:
ack